src/styles/themes/_default-theme.scss

```diff
@@ -712,4 +712,33 @@
   // ------ Spinner theme ------
   --spinner-color: var(--primary-color);
   // ------ Spinner theme ------
+
+  // ------ Slider theme ------
+  --slider-data-active-mark-one-background-color: var(--grey-color-30);
+  --slider-data-active-mark-one-border-color: var(--grey-color-60);
+  --slider-data-dot-background-color: var(--grey-color-10);
+  --slider-data-inactive-mark-one-background-color: var(--violet-color-30);
+  --slider-data-inactive-mark-one-border-color: var(--violetred-color-60);
+  --slider-data-inactive-mark-two-background-color: var(--blue-color-30);
+  --slider-data-inactive-mark-two-border-color: var(--blueviolet-color-60);
+  --slider-data-inactive-mark-three-background-color: var(--orange-color-30);
+  --slider-data-inactive-mark-three-border-color: var(--orange-color-60);
+  --slider-data-rail-background-color: var(--grey-color-30);
+  --slider-data-rail-border-color: var(--grey-color-60);
+  --slider-extremity-text-color: var(--grey-color-60);
+  --slider-benchmark-background-color: var(--grey-color-30);
+  --slider-benchmark-border-color: var(--grey-color-60);
+  --slider-rail-background-color: var(--accent-color-10);
+  --slider-rail-border-color: var(--primary-color-60);
+  --slider-thumb-color: var(--accent-color-20);
+  --slider-track-background-color: var(--primary-color-60);
+  --slider-track-border-color: var(--primary-color-60);
+  --slider-track-green-background-color: var(--green-color-30);
+  --slider-track-green-border-color: var(--green-color-60);
+  --slider-track-orange-background-color: var(--orange-color-30);
+  --slider-track-orange-border-color: var(--orange-color-60);
+  --slider-track-red-background-color: var(--red-color-30);
+  --slider-track-red-border-color: var(--red-color-60);
```

Comment:
recommend naming these according to status rather than color
slider-track-success, warning, error